Python/Product/Analysis/Analyzer/ExpressionEvaluator.cs

```diff
@@ -349,6 +349,16 @@ internal class ExpressionEvaluator {
                 return ee.ProjectState.GetAnalysisValueFromObjects(ee.ProjectState.GetTypeFromObject(n.Value)).GetInstanceType();
             }
 
+            // Members of class derived from Enum are Enums
+            if(ee.Scope is ClassScope cs) {
+                var baseEnum = cs.Class.Bases
+                    .OfType<MultipleMemberInfo>()
+                    .FirstOrDefault(mmi => mmi.Members.OfType<BuiltinClassInfo>().Any(x => x.Name == "Enum"));
```

Comment:
Yeah, that enum thing is a bit of a hack. I'll see what I can do here. Might as well separate into a different PR since issue is not very high priority and is broken in Jedi as well.

Reply:
Another alternative is to specifically check that base is `enum.Enum`. Probably bit simpler than wrapping the module.